packages/neo-one-node-vm/src/__tests__/opcodes.test.js

```diff
@@ -870,16 +914,30 @@ const OPCODES = ([
       result: [new UInt256StackItem(crypto.hash256(Buffer.alloc(32, 1)))],
       gas: FEES.TWENTY,
     },
-    // TODO (afragapane): Need to test true condition
+    {
+      op: 'CHECKSIG',
+      args: [keys[0].publicKey, signature0],
+      result: [new BooleanStackItem(true)],
+      gas: FEES.ONE_HUNDRED,
+    },
     {
       op: 'CHECKSIG',
       args: [keys[0].publicKey, Buffer.alloc(64, 10)],
       result: [new BooleanStackItem(false)],
       gas: FEES.ONE_HUNDRED,
     },
-    // {
-    //   op: 'CHECKMULTISIG',
-    // },
+    {
+      op: 'CHECKMULTISIG',
+      args: [[keys[0].publicKey, keys[1].publicKey], [signature0, signature1]],
```

Comment:
Can you test this where we have say 3 public keys and 2 signatures, where the first and third public key match the signatures?